dead_mans_switch_server/security/res_groups.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        <record id="group_user" model="res.groups">
+            <field name="name">User</field>
```

Comment:
My remark was to rename it as current name is not explicit in a list (same for manager) but if you think it is enough, I trust your judgement